mcp/mastra/verticals/calendar/tools.spec.ts

```diff
@@ -0,0 +1,76 @@
+// @ts-expect-error - bun:test types are built into Bun runtime
+import { beforeAll, describe, expect, it } from 'bun:test';
+import { getAllCalendars, getCalendarEvents } from './tools';
+
+// Type guard to check if result is a validation error
+function isValidationError(result: unknown): result is { error: true; message: string } {
+  return (
+    typeof result === 'object' && result !== null && 'error' in result && (result as { error: boolean }).error === true
+  );
+}
+
+describe('Calendar Tools Integration Tests', () => {
+  beforeAll(() => {
+    // Verify Google OAuth credentials are configured
+    const hasCredentials =
+      process.env.HEY_JARVIS_GOOGLE_CLIENT_ID &&
+      process.env.HEY_JARVIS_GOOGLE_CLIENT_SECRET &&
+      process.env.HEY_JARVIS_GOOGLE_REFRESH_TOKEN;
+
+    if (!hasCredentials) {
+      console.log('⚠️  Google OAuth credentials not configured - tests will validate error handling');
```

Comment:
Tests should never be conditional. Correct this for all tests. If some env values are missing, it should just fail.